frontend/packages/dev-console/src/components/formik-fields/ResourceLimitField.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import cx from 'classnames';
+import { useField, useFormikContext, FormikValues } from 'formik';
+import { FormGroup, ControlLabel, HelpBlock } from 'patternfly-react';
+import { NumberSpinner, Dropdown } from '@console/internal/components/utils';
+import { getValidationState } from './field-utils';
+import { ResourceLimitFieldProps } from './field-types';
+
+const ResourceLimitField: React.FC<ResourceLimitFieldProps> = ({
+  inputLabel,
+  unitLabel,
+  name,
+  unitName,
+  helpText,
+  ...props
+}) => {
+  const [field, { touched, error }] = useField(name);
+  const { setFieldValue, setFieldTouched } = useFormikContext<FormikValues>();
+
+  return (
+    <FormGroup controlId={`${name}-field`} validationState={getValidationState(error, touched)}>
+      <div className="row">
+        <div className="col-lg-4 col-md-4 col-sm-6 col-xs-7">
+          <ControlLabel className={cx({ 'co-required': props.required })}>
+            {inputLabel}
+          </ControlLabel>
+          <NumberSpinner
```

Comment:
You can use `NumberSpinnerField` here instead.

Reply:
Yes, I can use ```NumberSpinnerField``` but there is an issue in showing ```helpText```. In ```resourceLimitField``` we have to show the two field in two columns and a single ```helpText``` for both fields. That is why I combine ```NumberSpinner``` and ```Dropdown```` component and make a single field.